test/parallel/test-pipe-writev.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
+
+if (common.isWindows) {
+  common.skip('Unix-specific test');
+  return;
+}
+
+common.refreshTmpDir();
+
+function startServer() {
```

Comment:
...well, at least if this `startClient` function exists at all. I'll simplify the code so that it resolves this remark and the one related to `bind` at once.